WordPress/Classes/ViewRelated/Post/PostListHelper.swift

```diff
@@ -0,0 +1,33 @@
+import Foundation
+
+struct PostListHelper {
```

Comment:
While `openEditor` is directly related to the Post List I'm not sure the same applies to the date formatted. What do you think?

Any idea of how can we refactor that?

Reply:
The date formatter should really be in the other repo, but I'm not sure how to go about that. It was originally just stuck in the PostListEditorPresenter, so it wasn't in the right location in the first place.